config/jobs/etcd/etcd-postsubmits.yaml

```diff
@@ -5,6 +5,7 @@ postsubmits:
     cluster: eks-prow-build-cluster
     branches:
     - main
+    - release-3.6
```

Comment:
Question: why release-3.5 (and probably release-3.4) isn't included here?

Reply:
It's mostly the same answer from https://github.com/kubernetes/test-infra/pull/34287#discussion_r1946858138.

Currently, not all branches use the same make targets to run the tests. So, we'll need to backport some pull requests to standardize how we run tests and build the project.